_includes/configuration/valid-styles.md

```diff
@@ -1,6 +1,8 @@
 ## valid_styles
 
-This option enables you specify the available styles for each element. This means you can force ordering and only specific styles will be valid within style attribute values. Specify the element name, then it's styles or `*` for all elements.
+This option allows you to specify the available styles for each element. This option also forces the ordering of the styles. To use this option, specify the element name, then the allowed styles or `*` for all elements.
```

Comment:
This is a bit confusing to me, as is the asterisk referring to the element name or the styles list? I know it's the former but given it's directly after the mention of "styles" it sounds like the asterisk applies to the styles.

So may be something like this instead?
```
To use this option, specify an object containing a mapping of element names to allowed styles. To specify the allowed styles for all elements use `*` as the element name.
```

Reply:
fixed